frontend/pulse3d/src/pages/upload-form.js

```diff
@@ -135,6 +135,19 @@ export default function UploadForm() {
     setCheckedParams(newCheckedParams);
   };
 
+  useEffect(() => {
+    if (files.length > 0) {
+      const incompatible_names = [];
+      files.forEach((file) =>
+        file.name.includes("(") || file.name.includes(")") ? incompatible_names.push(file.name) : null
+      );
+      if (incompatible_names.length > 0) {
```

Comment:
You could filter the files for incompatible names to simplify:
`const incompatible_names = files.filter((file) => file.name.includes("(") || file.name.includes(")"));`

Reply:
So instead of prompting the user to reupload we just remove the conflicting files?

Reply:
we want to allow users to upload files with those characters since they are valid file names